src/actions/ServiceGraphActions.ts

```diff
@@ -0,0 +1,19 @@
+import { createAction } from 'typesafe-actions';
+import { CytoscapeClickEvent } from '../components/CytoscapeGraph/CytoscapeGraph';
+
+export enum ServiceGraphActionKeys {
+  GRAPH_OBJECT_CLICKED = 'GRAPH_OBJECT_CLICKED',
+  GRAPH_READY = 'GRAPH_READY'
```

Comment:
Remember that these redux events should self-describe themselves if one is reading the Redux action log. If one comes across a global event called 'GRAPH_READY' it doesn't tell me *how* this event is going to change the state and to which graph.

Reply:
The Redux events aren't namespaced and since they are global they need to be understandable on their own.